controllers/cloudstackisolatednetwork_controller.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Whoohoooo! Isolated networks can be entirely ignored if not in use!

Well, almost.

The CS machine controller still adds the machine to a load balancer if on an isolated network.

I actually plan to add either a load balancer CRD or a watch on this controller to get that out of the machine controller eventually. I really want other CRDs to be entirely ignorant of the type of network they're on--it's just not their responsibility.